scalatest/src/main/scala/org/scalatest/Entry.scala

```diff
@@ -48,21 +48,21 @@ case class Entry[K, V](key: K, value: V) extends java.util.Map.Entry[K, V] {
    *
    * @return the key corresponding to this entry
    */
-  def getKey: K = key
+  def getKey: K = key.asInstanceOf[K]
 
   /**
    * Returns the value corresponding to this entry.
    *
    * @return the value corresponding to this entry 
    */
-  def getValue: V = value
+  def getValue: V = value.asInstanceOf[V]
 
   /**
    * Throws <code>UnsupportedOperationException</code>.
    *
    * @throws UnsupportedOperationException unconditionally
    */
-  def setValue(v: V): V = throw new UnsupportedOperationException
+  def setValue(v: V | Null): V = throw new UnsupportedOperationException
```

Comment:
**Overridden java method arguments are nullable**
A Scala method overrides some Java method. We need to add `|Null` to the method's parameter types to make the override still work.